cygnus-ngsi/pom.xml

```diff
@@ -1,165 +1,204 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
It seems that some pretty printer has ruineed the whole pom.xml :(

Could you come back to the original one in master adding just the elements needed by the new sink so the PR can clearly show the modifications?

Reply:
I changed the pom.xml and I added new library and repository.